server/flaskr/database/postgres/postgres.py

```
@@ -12,14 +14,40 @@ def close_db(e=None):
+    with get_db_access() as conn:
+        cur = conn.cursor()
+
+        query = "INSERT INTO public.organizations (slug, name) values (%s, %s);"
+        params = ("test", "Test")
+        cur.execute(query, params)
+
+        cur.execute("CREATE SCHEMA IF NOT EXISTS test;")
+        cur.execute("SET search_path TO test, public;")
+
+        with current_app.open_resource(
+            os.path.join("database", "postgres", f"schema.sql")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)